Description: tests time out on builders, which are often busy or sometimes on
slower hardware. Tests fail on riscv64 builder.
Upstream: No
diff --git a/pkg/kapp/diffgraph/change_graph_cf_for_k8s_test.go b/pkg/kapp/diffgraph/change_graph_cf_for_k8s_test.go
index 5f4c551..c22b5e4 100644
--- a/pkg/kapp/diffgraph/change_graph_cf_for_k8s_test.go
+++ b/pkg/kapp/diffgraph/change_graph_cf_for_k8s_test.go
@@ -37,7 +37,7 @@ func TestChangeGraphCFForK8sUpsert(t *testing.T) {
 	graph, err := buildChangeGraphWithOpts(opts, t)
 	require.NoErrorf(t, err, "Expected graph to build")
 
-	require.Less(t, time.Now().Sub(t1), time.Duration(1*time.Second), "Graph build took too long")
+	require.Less(t, time.Now().Sub(t1), time.Duration(2*time.Second), "Graph build took too long")
 
 	output := strings.TrimSpace(graph.PrintLinearizedStr())
 	expectedOutput := strings.TrimSpace(cfForK8sExpectedOutputUpsert)
@@ -67,7 +67,7 @@ func TestChangeGraphCFForK8sDelete(t *testing.T) {
 	graph, err := buildChangeGraphWithOpts(opts, t)
 	require.NoErrorf(t, err, "Expected graph to build")
 
-	require.Less(t, time.Now().Sub(t1), time.Duration(1*time.Second), "Graph build took too long")
+	require.Less(t, time.Now().Sub(t1), time.Duration(2*time.Second), "Graph build took too long")
 
 	output := strings.TrimSpace(graph.PrintLinearizedStr())
 	expectedOutput := strings.TrimSpace(cfForK8sExpectedOutputDelete)
